webpack/modules/aws-cdk-lib/aws-lambda-nodejs/lib/esbuild.js

```diff
@@ -0,0 +1,45 @@
+// Load the go wasm.
+require("./go-wasm");
+
+class EsBuild extends Go {
+  constructor() {
+    super();
+    this.wasm = undefined;
+    this.workDir = "/app";
```

Comment:
shouldn't this be `process.env.CDK_OUTDIR` ?

Reply:
No, `/app` was correct. It should be `cwd` but it's actually unused here now. Removing it.